tools/AndroidDependencies/Assets/UnityAndroidDependenciesSetter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
```

Comment:
should we move this to the `Editor` folder so that it will automatically be excluded for builds? 
Sure, people are not supposed to build it, but if we ever extend that project for any reason

Reply:
good idea